projects/RabbitMQ.Client/client/impl/OutgoingCommand.cs

```diff
@@ -40,39 +40,35 @@
 
 using System;
 using System.Buffers;
-using System.Runtime.InteropServices;
 using RabbitMQ.Client.Framing.Impl;
 
 namespace RabbitMQ.Client.Impl
 {
-    class Command : IDisposable
+    internal readonly struct OutgoingCommand
```

Comment:
```
Marshal.SizeOf(typeof(OutgoingCommand));
```

shows it is 32 bytes large which technically violates the principle of structs should be around 16-24 bytes. Am I assuming correctly the potential perf impact of that is largely mitigated by passing them around with `in`,`out` or `ref`?

Reply:
Also here, yes the `in` modifier neglects it.